data-publisher/src/main/kotlin/no/nav/amt/tiltak/data_publisher/publish/DeltakerlistePublishQuery.kt

```diff
@@ -18,6 +18,10 @@ class DeltakerlistePublishQuery(
 ) {
 
 	fun get(id: UUID): DeltakerlistePublishDto? {
+		if(IgnoredDeltakerlister.deltakerlisteIds.contains(id)) {
```

Comment:
Jeg ville ha flyttet denne sjekken opp til `publishDeltakerliste` i DataPublisherService for at koden skal bli lettere å følge. Jeg trodde først sjekken var glemt, for jeg så ikke at denne logikken lå i noe som jeg bare trodde satte sammen et objekt. Det samme gjelder egentlig for deltaker også. 

Du kan velge selv om du vil endre det for jeg tror koden gjør det den skal, men den er litt vanskelig å lese.